clients/drcachesim/tools/basic_counts.cpp

```diff
@@ -282,3 +282,15 @@ basic_counts_t::print_results()
 
     return true;
 }
+
+basic_counts_t::counters_t
+basic_counts_t::get_total_counts()
```

Comment:
Hmm, xref #3068

Reply:
Added a comment and to the PR description.

Reply:
https://github.com/DynamoRIO/dynamorio/pull/5720.